### PR TITLE
Feat/126 rust language bindings

### LINKED ISSUE #126: Create Rust language bindings from Fluctuo's model
## Context
Fluctuo has been gracious enough to suggest we can use their pre-existing codebase for this: [fluctuo/gbfs_types](https://github.com/fluctuo/gbfs_types). The library is hosted on https://crates.io/crates/gbfs_types. 

The objective is to increase the visibility of tools that improve the quality of GBFS feeds.

## To do
Work with @tdelmas to:
- [ ] Merge [fluctuo/gbfs_types](https://github.com/fluctuo/gbfs_types) into [MobilityData/gbfs-json-schema](https://github.com/MobilityData/gbfs-json-schema).
- [ ] Add MobilityData's GitHub team as a co-owner of https://crates.io/crates/gbfs_types/.
- [ ] Update the model to support v3.0 (@tdelmas may be able to help here).

The process should be similar to https://github.com/MobilityData/gbfs-json-schema/issues/109/.

### PR DESCRIPTION
**Pull request by Alessandro100**
Fixes #126 

This PR merges the repo https://github.com/fluctuo/gbfs_types big thanks to @tdelmas 🎉

Whats been done in this PR

- updated readme
- updated files from v3.0-rc2 to v3.0

What's next
- CICD configuration
- publishing

### CODE REVIEW

**Review by tdelmas — APPROVED** [review]

File: models/rust/src/3.0/files/system_pricing_plans.rs [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    /// In case of rate price, this field is the base price that is charged only once per trip (typically the price for unlocking) in addition to per_km_pricing and/or per_min_pricing.
+    pub price: Option<NonNegativeFloat>,
+    /// The cost, described as per minute rate, to reserve the vehicle prior to beginning a rental.
+    /// This amount is charged for each minute of the vehicle reservation until the rental is initiated, or until the number of minutes defined in vehicle_types.json#default_reserve_time elapses, whichever comes first.
+    /// This field MUST NOT be combined in a single pricing plan with reservation_price_flat_rate.
+    pub reservation_price_per_min: Option<NonNegativeFloat>,
+    /// The cost, described as a flat rate, to reserve the vehicle prior to beginning a rental.
+    /// This amount is charged once to reserve the vehicle for the duration of the time defined by vehicle_types.json#default_reserve_time.
+    /// This field MUST NOT be combined in a single pricing plan with reservation_price_per_min.
+    pub reservation_price_flat_rate: Option<NonNegativeFloat>,
```
Comment (richfab) [inline models/rust/src/3.0/files/system_pricing_plans.rs:47]: These 2 fields were removed from v3.0 and added to v3.1-RC ([Release Notes](https://github.com/MobilityData/gbfs/releases/tag/v3.1-RC)).

```suggestion
```

File: models/rust/src/3.0/files/examples/specification/system_pricing_plans-1.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                "plan_id": "plan2",
+                "name": [
+                    {
+                        "text": "One-Way",
+                        "language": "en"
+                    }
+                ],
+                "currency": "USD",
+                "price": 2.00,
+                "reservation_price_per_min": 0.15,
```
Comment (richfab) [inline models/rust/src/3.0/files/examples/specification/system_pricing_plans-1.json:17]: This field was removed from v3.0 and added to v3.1-RC ([Release Notes](https://github.com/MobilityData/gbfs/releases/tag/v3.1-RC)).

```suggestion
```

**Review by richfab — CHANGES_REQUESTED** [review]
Thank you Fluctuo (@tdelmas) for this 🙏 🙏 🙏 I'm sure this library will benefit many people in the community.

Nice work @Alessandro100. I left one comment.

**Review by richfab — APPROVED** [review]
LGTM from reading through the files.

It would be good that a dev from MobilityData reviews this PR 🙏

**Review by davidgamez — APPROVED** [review]
Great addition to the language bindings! 🥳

**Pull request merged**